WheelOfFortune/Game.cs

```diff
@@ -50,43 +50,29 @@ public class Game
         /// </summary>
         public void StartGame()
         {
-            Console.WriteLine("Press any button to start the game!");
-            Console.ReadKey(true);
-            AddPlayer();
+            Prompt.WelcomeMessage();
 
-            Random random = new();
-            int index = random.Next(0, allPuzzles.Count);
-            string nextPuzzle = allPuzzles[index];
-            Puzzle puzzle = new(nextPuzzle);
-            CurrentPuzzle = puzzle;
+            string person = Prompt.CreatePlayer();
+            Player player = new Player(person);
```

Comment:
Same here - would it be better for the variable to be something like `newPlayer`?

Reply:
Changed